pyomo/dae/tests/test_misc.py

```diff
@@ -484,7 +484,7 @@ def _con2(_b, i, ti):
 
         generate_finite_elements(model.t, 5)
 
-        missing_idx = set(model.blk._index) - set(model.blk._data.keys())
+        missing_idx = set(model.blk._index_set) - set(model.blk._data.keys())
```

Comment:
Shouldn't this (and the subsequent 3 instances) all use the public `index_set()` method?